src/lib/components/View.svelte

```diff
@@ -64,7 +64,6 @@
     if (state.error !== undefined) {
       error = true;
       errorLines = state.error.toString().split('\n');
-      return;
     }
     error = false;
```

Comment:
If return is removed, error is set as false immediately. So something else needs to be changed.